wire-ios-sync-engine/Tests/Source/Calling/MLSConferenceStaleParticipantsRemoverTests.swift

```diff
@@ -150,8 +152,7 @@ class MLSConferenceStaleParticipantsRemoverTests: MessagingTest {
         }
 
         // set expectations
-        let expectation = XCTestExpectation()
-        expectation.isInverted = true
+        let expectation = InvertedTestExpectation()
```

Comment:
Hmm I prefer the Apple version, but it's personal taste..

Reply:
Changed it, what do you think now?
53358e05f9adb868b1042078d233044b2a3c0376